Project.toml

```
@@ -15,7 +15,6 @@ UnitfulAstro = "6112ee07-acf9-5e0f-b108-d242c714bf9f"
 [compat]
 DataDeps = "0.7"
```

Looks like the last version of BSplineKit.jl to support Julia 1.6 (our current bound) was v0.11 -- see [here](https://github.com/jipolanco/BSplineKit.jl/compare/v0.11.0...master). The CI run on Julia 1.6 should let us know if this works or not.

```suggestion
BSplineKit = "0.11"
DataDeps = "0.7"
```

---

Thanks for checking this. I have next to no experience with anything versioning related